ODS CI build breaks due to qs ds-rshiny missing dependency

ODS CI build is failing due to a missing dependency in `ds-rshiny`.

It looks like because this missing dependency:
```
ESC[0;32m    amazon-ebs: error: build error: The command '/bin/sh -c apt-key adv --fetch-keys http://archive.cloudera.com/cdh5/ubuntu/xenial/amd64/cdh/archive.key &&     echo 'deb [arch=amd64] http://archive.clo
udera.com/cdh5/ubuntu/xenial/amd64/cdh xenial-cdh5 contrib' > /etc/apt/sources.list.d/cloudera.list' returned a non-zero code:
```

I get a 404 response if I try to open in the browser `http://archive.cloudera.com/cdh5/ubuntu/xenial/amd64/cdh/archive.key` 

**Log Output**
```
ESC[0;32m    amazon-ebs: Executing: /tmp/tmp.gu4p8uXW2T/gpg.1.sh --keyserverESC[0m
ESC[0;32m    amazon-ebs: hkp://keyserver.ubuntu.com:80ESC[0m
ESC[0;32m    amazon-ebs: --recv-keysESC[0m
ESC[0;32m    amazon-ebs: E298A3A825C0D65DFD57CBB651716619E084DAB9ESC[0m
ESC[0;32m    amazon-ebs: ESC[91mgpg: requesting key E084DAB9 from hkp server keyserver.ubuntu.comESC[0m
ESC[0;32m    amazon-ebs: ESC[0mESC[91mgpg: key E084DAB9: public key "Michael Rutter <marutter@gmail.com>" importedESC[0m
ESC[0;32m    amazon-ebs: gpg: Total number processed: 1ESC[0m
ESC[0;32m    amazon-ebs: gpg:               imported: 1  (RSA: 1)ESC[0m
ESC[0;32m    amazon-ebs: ESC[0mRemoving intermediate container 014663a2aed8ESC[0m
ESC[0;32m    amazon-ebs:  ---> 7695af124f64ESC[0m
ESC[0;32m    amazon-ebs: Step 10/20 : RUN apt-key adv --fetch-keys http://archive.cloudera.com/cdh5/ubuntu/xenial/amd64/cdh/archive.key &&     echo 'deb [arch=amd64] http://archive.cloudera.com/cdh5/ubuntu/xenia
l/amd64/cdh xenial-cdh5 contrib' > /etc/apt/sources.list.d/cloudera.listESC[0m
ESC[0;32m    amazon-ebs:  ---> Running in 921b4b35163eESC[0m
ESC[0;32m    amazon-ebs: Executing: /tmp/tmp.v6YX2IV3Gw/gpg.1.sh --fetch-keysESC[0m
ESC[0;32m    amazon-ebs: http://archive.cloudera.com/cdh5/ubuntu/xenial/amd64/cdh/archive.keyESC[0m
ESC[0;32m    amazon-ebs: gpgkeys: no key data found for http://archive.cloudera.com/cdh5/ubuntu/xenial/amd64/cdh/archive.keyESC[0m
ESC[0;32m    amazon-ebs: ESC[91mgpg: no valid OpenPGP data found.ESC[0m
ESC[0;32m    amazon-ebs: gpg: Total number processed: 0ESC[0m
ESC[0;32m    amazon-ebs: gpg: keyserver communications error: key not foundESC[0m
ESC[0;32m    amazon-ebs: gpg: keyserver communications error: bad public keyESC[0m
ESC[0;32m    amazon-ebs: gpg: WARNING: unable to fetch URI http://archive.cloudera.com/cdh5/ubuntu/xenial/amd64/cdh/archive.key: bad public keyESC[0m
ESC[0;32m    amazon-ebs: ESC[0mRemoving intermediate container 921b4b35163eESC[0m
ESC[0;32m    amazon-ebs: error: build error: The command '/bin/sh -c apt-key adv --fetch-keys http://archive.cloudera.com/cdh5/ubuntu/xenial/amd64/cdh/archive.key &&     echo 'deb [arch=amd64] http://archive.clo
udera.com/cdh5/ubuntu/xenial/amd64/cdh xenial-cdh5 contrib' > /etc/apt/sources.list.d/cloudera.list' returned a non-zero code: 2ESC[0m
ESC[0;32m    amazon-ebs: [Pipeline] }ESC[0m
ESC[0;32m    amazon-ebs: [Pipeline] // timeoutESC[0m
ESC[0;32m    amazon-ebs: [Pipeline] shESC
ESC[0;32m    amazon-ebs: WARNING: Unknown parameter(s) found for class type 'org.jenkinsci.plugins.workflow.steps.durable_task.ShellStep': labelESC[0m
ESC[0;32m    amazon-ebs: + oc -n unitt-cd get build rshiny-1 -o 'jsonpath={.status.phase}'ESC[0m
ESC[0;32m    amazon-ebs: [Pipeline] sleepESC[0m
ESC[0;32m    amazon-ebs: Sleeping for 12 secESC[0m
ESC[0;32m    amazon-ebs: [Pipeline] shESC[0m
ESC[0;32m    amazon-ebs: WARNING: Unknown parameter(s) found for class type 'org.jenkinsci.plugins.workflow.steps.durable_task.ShellStep': labelESC[0m
ESC[0;32m    amazon-ebs: + oc -n unitt-cd get build rshiny-1 -o 'jsonpath={.status.phase}'ESC[0m
ESC[0;32m    amazon-ebs: [Pipeline] sleepESC[0m
ESC[0;32m    amazon-ebs: Sleeping for 12 secESC[0m
ESC[0;32m    amazon-ebs: [Pipeline] shESC[0m
ESC[0;32m    amazon-ebs: WARNING: Unknown parameter(s) found for class type 'org.jenkinsci.plugins.workflow.steps.durable_task.ShellStep': labelESC[0m
ESC[0;32m    amazon-ebs: + oc -n unitt-cd get build rshiny-1 -o 'jsonpath={.status.phase}'ESC[0m
ESC[0;32m    amazon-ebs: [Pipeline] sleepESC[0m
ESC[0;32m    amazon-ebs: Sleeping for 12 secESC[0m
ESC[0;32m    amazon-ebs: [Pipeline] errorESC[0m
ESC[0;32m    amazon-ebs: [Pipeline] }ESC[0m
ESC[0;32m    amazon-ebs: [Pipeline] // stageESC[0m
ESC[0;32m    amazon-ebs: [Pipeline] echoESC[0m
ESC[0;32m    amazon-ebs: [Build OpenShift Image-rshiny] **** ENDED stage 'Build OpenShift Image' for component 'rshiny' branch 'master' **** (took 103507 ms)ESC[0m
ESC[0;32m    amazon-ebs: [Pipeline] }ESC[0m
ESC[0;32m    amazon-ebs: [Pipeline] // wrapESC[0m
ESC[0;32m    amazon-ebs: [Pipeline] stageESC[0m
ESC[0;32m    amazon-ebs: [Pipeline] { (odsPipeline error)ESC[0m
ESC[0;32m    amazon-ebs: [Pipeline] echoESC[0m
ESC[0;32m    amazon-ebs: WARN: [rshiny] ***** Finished ODS Pipeline for rshiny (with error) ***** (took 142652 ms)ESC[0m
ESC[0;32m    amazon-ebs: [Pipeline] echoESC[0m
ESC[0;32m    amazon-ebs: WARN: Error: hudson.AbortException: OpenShift Build #1 was not successful - status is 'failed'.ESC[0m
ESC[0;32m    amazon-ebs: [Pipeline] withCredentialsESC[0m
ESC[0;32m    amazon-ebs: Masking supported pattern matches of $USERNAME or $TOKENESC[0m
ESC[0;32m    amazon-ebs: [Pipeline] {ESC[0m
ESC[0;32m    amazon-ebs: [Pipeline] shESC[0m
ESC[0;32m    amazon-ebs: WARNING: Unknown parameter(s) found for class type 'org.jenkinsci.plugins.workflow.steps.durable_task.ShellStep': labelESC[0m
ESC[0;32m    amazon-ebs: + curl --fail -sS --request POST --header 'Authorization: Bearer ****' --header 'Content-Type: application/json' --data '{"state":"FAILED","key":"065c7ffa","name":"065c7ffa","url":"https
://jenkins-unitt-cd.ocp.odsbox.lan/job/unitt-cd/job/unitt-cd-run-unitt-rshiny-unitt-master/1/"}' http://bitbucket.odsbox.lan:7990/rest/build-status/1.0/commits/065c7ffa7574a2a984f1e6b4c541f421613efeaeESC[0m
ESC[0;32m    amazon-ebs: [Pipeline] }E


[0;32m    amazon-ebs:     --- FAIL: TestQuickstarter/ds-rshiny (263.06s)
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,5 +1,9 @@
 # Changelog
 
+## Unreleased
+
+- ds-rshiny cleanup cloudera dependency ([#540](https://github.com/opendevstack/ods-quickstarters/pull/540))
+
 ## [3.0] - 2020-08-11
 
 ### Added

--- a/ds-rshiny/files/docker/Dockerfile
+++ b/ds-rshiny/files/docker/Dockerfile
@@ -26,10 +26,6 @@ ENV LANG en_US.UTF-8
 RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys E298A3A825C0D65DFD57CBB651716619E084DAB9 && \
     echo 'deb [arch=amd64,i386] https://cran.rstudio.com/bin/linux/ubuntu xenial/' > /etc/apt/sources.list.d/cran.list
 
-# Add Cloudera apt repo
-RUN apt-key adv --fetch-keys http://archive.cloudera.com/cdh5/ubuntu/xenial/amd64/cdh/archive.key && \
-    echo 'deb [arch=amd64] http://archive.cloudera.com/cdh5/ubuntu/xenial/amd64/cdh xenial-cdh5 contrib' > /etc/apt/sources.list.d/cloudera.list
-
 # Install dependencies
 RUN apt-get update && \
     apt-get install -yqq \
@@ -39,7 +35,6 @@ RUN apt-get update && \
         libxt-dev \
         pandoc \
         pandoc-citeproc \
-        hive-jdbc \
         r-base \
         r-base-dev \
         r-cran-rjava

